akka-stream/src/main/scala/akka/stream/impl/fusing/IteratorInterpreter.scala

```diff
@@ -145,7 +145,8 @@ private[akka] class IteratorInterpreter[I, O](val input: Iterator[I], val ops: S
       outHandlers,
       logics,
       (_, _, _) ⇒ throw new UnsupportedOperationException("IteratorInterpreter does not support asynchronous events."),
-      fuzzingMode = false)
+      fuzzingMode = false,
+      null)
```

Comment:
(this part of the code referred to the interpreter)
It requires analyzing the full code to then deduce the lifecycle (when it has a value which doesn't blow up, and if it is appropriately guarded wherever needed).